app.json

```diff
@@ -87,6 +87,9 @@
     "SIGNUP": {
       "required": true
     },
+    "SECRET_COOKIE_PASSWORD": {
+      "required": true
```

Comment:
lets add `"generator": "secret"` to this, it doesn't need to inherit this secret from the staging environment.

Reply:
Are there other places we can us the generator, so that we're not re-using staging secrets?

Reply:
good catch!

Reply:
don't think there are other instances